### PR TITLE
Replace state with boolean

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-react/src/ui/commons/WaitingForApp.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const WaitingForApp = ({description, className}: WaitingForProps) => (
 
   <div className={useClassFor('waitingfortransaction')}>
     <WaitingFor
-      action="Configure UniLogin application"
+      action="Loading UniLogin application"
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/WaitingForApp.tsx]: `Loading...`

File: universal-login-web3/src/services/UIController.ts [hunk 2]
```diff
@@ -84,10 +83,10 @@ export class UIController {
   }
 
   appInitialization() {
```
Comment (Marik-D) [inline universal-login-web3/src/services/UIController.ts]: Is this needed considering `isLoading === true` by default?

Comment (xitronix) [reply universal-login-web3/src/services/UIController.ts]: it is not neccessary

File: universal-login-web3/src/ui/react/ULWeb3Root.tsx [hunk 3]
```diff
@@ -18,15 +18,16 @@ export const ULWeb3Root = ({sdk, walletService, uiController, domains}: ULWeb3Ro
   const modal = useProperty(uiController.activeModal);
   const message = sdk.getNotice();
 
+  if (uiController.isLoading.get()) {
```
Comment (Marik-D) [inline universal-login-web3/src/ui/react/ULWeb3Root.tsx]: This won't update the react component. Use `usePropperty` instead

```suggestion
  if (useProperty(uiController.isLoading)) {
```

File: universal-login-web3/src/ui/react/ULWeb3Root.tsx [hunk 4]
```diff
@@ -18,15 +18,16 @@ export const ULWeb3Root = ({sdk, walletService, uiController, domains}: ULWeb3Ro
   const modal = useProperty(uiController.activeModal);
   const message = sdk.getNotice();
 
+  if (useProperty(uiController.isLoading)) {
+    return <ModalWrapper><WaitingForApp /></ModalWrapper>;
```
Comment (Marik-D) [inline universal-login-web3/src/ui/react/ULWeb3Root.tsx]: Rename to `AppPreloader`

File: universal-login-react/src/ui/commons/AppPreloader.tsx [hunk 5]
```diff
@@ -6,11 +6,11 @@ import '../styles/themes/Legacy/waitingForTransactionThemeLegacy.sass';
 import '../styles/themes/Jarvis/waitingForTransactionThemeJarvis.sass';
 import '../styles/themes/UniLogin/waitingForTransactionThemeUniLogin.sass';
 
-export const WaitingForApp = ({description, className}: WaitingForProps) => (
+export const AppPreloader = ({description, className}: WaitingForProps) => (
 
   <div className={useClassFor('waitingfortransaction')}>
     <WaitingFor
-      action="Configure UniLogin application"
+      action="AppPreloader..."
```
Comment (Marik-D) [inline universal-login-react/src/ui/commons/AppPreloader.tsx]:
```suggestion
      action="Loading..."
```

**Pull request merged**